src/main/java/org/opensearch/performanceanalyzer/commons/metrics/AllMetrics.java

```diff
@@ -1037,6 +1041,12 @@ public static class Constants {
             public static final String SEARCHBP_SEARCH_BACK_PRESSURE_STATS_MODE = "mode";
             public static final String SEARCHBP_SEARCH_BACK_PRESSURE_STATS_SEARCH_TASK_STATS =
                     "searchTaskStats";
+
+            // General Info 
+            // Table Name stored in sqlitedb
+            public static final String SEARCHBP_TABLE_NAME = "Searchbp_Stats";
+            // Column Names for difference stats stored in sqlitedb
+            public static final String SEARCHBP_TYPE_DIM = "SearchBackPressureStats";
```

Comment:
Can we add some details about the purpose of this ?

Reply:
Resolved in latest commit.
Also paste here for better visibility.
```
Add SEARCHBP_TABLE_NAME for searchbackpressureRCA to find the corresponding table
Add SEARCHBP_TYPE_DIM for searchbackpressureRCA to find the cell value for each stats type
```

Reply:
@CoderJeffrey can you plz add comments for 2 constants separately?
e.g.
```
            /**
             * Add SEARCHBP_TABLE_NAME for searchbackpressureRCA to find the corresponding table
             */
            public static final String SEARCHBP_TABLE_NAME = "Searchbp_Stats";
            /**
            * Add SEARCHBP_TYPE_DIM for searchbackpressureRCA to find the cell value for each stats type
            * /
            public static final String SEARCHBP_TYPE_DIM = "SearchBackPressureStats";
```

Reply:
Latest commit added comments for 2 constants separately. Resolved.